gunicorn/workers/ggevent.py

```diff
@@ -143,13 +143,15 @@ def run(self):
 
             # Force kill all active the handlers
             self.log.warning("Worker graceful timeout (pid:%s)" % self.pid)
-            [server.stop(timeout=1) for server in servers]
+            for server in servers:
+                server.stop(timeout=1)
         except:
             pass
 
-    def handle_request(self, *args):
+    def handle_request(self, listener_name, req, sock, addr): #arguments-differ
```

Comment:
This comment is a bit cryptic. Arguments differ from base Worker class or Sync or Async? It looks like base class of this worker is Async and it has the same signature as this one:

```py
def handle_request(self, listener_name, req, sock, addr):
    ...
```

So perhaps we don't need a comment after all?